server/controllers/session-auth.js

```diff
@@ -0,0 +1,96 @@
+const {
+  fetchSpotifyToken,
+  fetchTwitchToken,
+  fetchTwitchChannel,
+} = require("../auth");
+
+const {
+  spotifyClientId,
+  spotifyClientSecret,
+  twitchClientSecret,
+  twitchClientId,
+} = require("../config/credentials.json");
```

Comment:
If we're putting it in `./server/config`, can we update the demo to match, and also update the documentation to reflect it?

Reply:
will do